src/main/java/com/github/cukespace/arquillian/asciidoctor/AsciidoctorObserver.java

```diff
@@ -43,20 +44,65 @@
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Map;
+import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.regex.Pattern;
 
 import static java.util.Arrays.asList;
 
 // fork of asciidoctor maven plugin
 public class AsciidoctorObserver {
+    
+    private static final Logger LOGGER = Logger.getLogger(AsciidoctorObserver.class.getName());
```

Comment:
can you remove it by default (= keep the laziness of it through the method)?

goal is to avoid to trigger JUL init until it is really needed to let the test framework set it up programmatically a bit later.